Wikipedia/Code/WMFArticleJSONCompilationHelper.m

```diff
@@ -0,0 +1,104 @@
+#import "WMFArticleJSONCompilationHelper.h"
+#import <WMF/MWKArticle.h>
+#import <WMF/NSString+WMFHTMLParsing.h>
+#import <WMF/WMFImageTag.h>
+#import <WMF/WMFImageTag+TargetImageWidthURL.h>
+#import <WMF/NSURL+WMFSchemeHandler.h>
+#import <WMF/MWKSection.h>
+#import <WMF/MWKSectionList.h>
+
+@implementation WMFArticleJSONCompilationHelper
+
++ (nullable NSData *)jsonDataForArticle: (MWKArticle *)article withImageWidth: (NSInteger)imageWidth {
```

Comment:
I figured this code will probably go away once we move to mobile-html, so I didn't see much point in converting it to Swift. It's from WMFURLSchemeHandler, I just moved it over into a separate helper class.